server/src/repositories/metadata.repository.ts

```diff
@@ -21,39 +21,41 @@ export class MetadataRepository implements IMetadataRepository {
   ) {
     this.logger.setContext(MetadataRepository.name);
   }
+  private exiftool: ExifTool = this.initExiftool();
 
   async teardown() {
-    await exiftool.end();
+    await this.exiftool.end();
+  }
+
+  private initExiftool() {
```

Comment:
Can you just inline this in the constructor instead?

Reply:
@jrasm91 Do you mean like `private exiftool: ExifTool = new ExifTool...` above?